IABot/checkIfDead.php

```diff
@@ -88,33 +87,14 @@ public function checkDeadlinks( $urls ) {
 		return $returnArray;
 	}
 
+
 	/**
-	 * Function to check whether a given link is dead
-	 * @param string $url URL of the link to be checked
-	 * @return bool True if dead; False if not
-	 * @access public
-	 * @author Niharika Kohli (Niharika29)
```

Comment:
I'd like to keep author attribution as well as access types.  Your name should be there, since you are its author.